x-pack/plugins/lens/public/app_plugin/app.tsx

```diff
@@ -89,6 +89,7 @@ export function App({
       isSaveModalVisible: false,
       indicateNoData: false,
       isSaveable: false,
+      searchSessionId: data.search.session.start(),
```

Comment:
Can you remove `dateRange` from the `LensAppState` now?

Reply:
Moved the `dateRange` out of state in the last commit. An object reference needs to be still hold for the `frame` component, but there's a single source of truth now for it.